tests/acceptance/conftest.py

```diff
@@ -56,11 +56,21 @@ def pytest_configure(config):
     """
     )
 
-    subprocess.call(
-        ["yarn", "webpack"],
-        env={
-            "NODE_ENV": "development",
-            "PATH": os.environ["PATH"],
-            "NODE_OPTIONS": "--max-old-space-size=4096",
-        },
-    )
+    try:
+        status = subprocess.call(
+            ["yarn", "--silent", "webpack"],
+            env={
+                "NODE_ENV": "development",
+                "PATH": os.environ["PATH"],
+                "NODE_OPTIONS": "--max-old-space-size=4096",
+            },
+        )
+
+        if status == 1:
```

Comment:
Err sorry, wait @billyvg you should `status != 0` here.

Reply:
```suggestion
        if status != 0:
```